datalad_crawler/nodes/annex.py

```diff
@@ -820,11 +822,11 @@ def merge_branch(data):
 
             if one_commit_at_a_time:
                 all_to_merge = list(
-                    self.repo.get_branch_commits(
+                    get_branch_commits(
+                        self.repo,
                         branch,
                         limit='left-only',
-                        stop=last_merged_checksum,
-                        value='hexsha'))[::-1]
+                        stop=last_merged_checksum))[::-1]
```

Comment:
with [recommended partial revert of the changes in datalad core PR](https://github.com/datalad/datalad/pull/3834/files#r353319087) the change here would be just to provide or not `value` depending on datalad version.